app/assets/javascripts/quby/answers/validation.js

```diff
@@ -80,13 +83,16 @@
                   }
                   break;
               case "maximum":
-                  if (inputs.length == 3 && (values[0] != "" && values[1] != "" && values[2] != "")) {
-                      var enteredDate = new Date(values[2], parseInt(values[1]) - 1, values[0]);
-                      var maximumDate = Date.parse(validation.value);
-
-                      if(enteredDate > maximumDate) {
+                  if ($(inputs[0]).attr('class') === 'date') {
+                    try {
+                      if(allDateFieldsFilledIn(inputs)) {
+                        var enteredDate = parsePartialDate(inputs);
+                        var maximumDate = new Date(validation.value)
+                        if(enteredDate > maximumDate) {
                           pushFailVal(validation.type);
+                        }
                       }
+                    } catch(e) {}
```

Comment:
Het is wel mijn bedoeling dat de validaties "minimum" en "maximum" worden overgeslagen als er een fout optreedt bij het parsen van de Date.

Stel een gebruiker voert de datum `01-taart-2015` in. Dan wil je de gebruiker laten weten dat het geen geldige datum is, en niet dat het niet in de juiste range valt.

Of kun je een andere situatie bedenken waarin er een exception wordt gecatcht, maar de gebruiker geen ongeldige datum heeft ingevuld?